index.html

```diff
@@ -9,61 +9,137 @@
-                <div class="card-body">
-                    <h5 class="card-title">Counter</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Counter/index.html" class="btn btn-primary">Visit</a>
+        </div>
+    </section>
+
+
+    <section id="project">
+        <div class="container my-3 text-center">
+            <h1> Awesome Javascript Projects</h1>
+        </div>
+        <div class="container border py-4">
+            <div class="row row-cols-3">
+                <div class="card m-3" style="width: 21rem;">
+                    <img class="card-img-top" src="Images/Project1.PNG" alt="Card image cap">
+                    <div class="card-body">
+                        <h5 class="card-title">Note app</h5>
+                        <!-- <p class="card-text">Some quick example text to build on the card title and make up the bulk of
+                            the
+                            card's content.</p> -->
+                        <a href="./public/note-app.html" class="btn btn-primary">Visit</a>
+                    </div>
+                </div>
+
+                <div class="card m-3" style="width: 21rem;">
+                    <img class="card-img-top" src="Images/Project2.PNG" alt="Card image cap">
+                    <div class="card-body">
+                        <h5 class="card-title">Calculator</h5>
+                        <!-- <p class="card-text">Some quick example text to build on the card title and make up the bulk of
+                            the
+                            card's content.</p> -->
+                        <a href="./public/calculator.html" class="btn btn-primary">Visit</a>
+                    </div>
+                </div>
+
+                <div class="card m-3" style="width: 21rem;">
+                    <img class="card-img-top" src="Images/Project3.PNG" alt="Card image cap">
+                    <div class="card-body">
+                        <h5 class="card-title">Color Flipper</h5>
+                        <!-- <p class="card-text">Some quick example text to build on the card title and make up the bulk of
+                            the
+                            card's content.</p> -->
+                        <a href="./public/color-flipper.html" class="btn btn-primary">Visit</a>
+                    </div>
                 </div>
-            </div>
 
-            <div class="card m-3" style="width: 18rem;">
-                <div class="card-body">
-                    <h5 class="card-title">Reviews</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Reviews/index.html" class="btn btn-primary">Visit</a>
+                <div class="card m-3" style="width: 21rem;">
```

Comment:
inline css not required.

Reply:
This inline css was of cards you picked.

Reply:
yep !! We can remove this now !!